src/entities/conditional-logic/model/conditions.ts

```diff
@@ -74,3 +78,226 @@ export const doesNotIncludeValue = <TItem>(
 
   return !input.includes(value);
 };
+
+export const isGreaterThanDate = (input: string, date: string) => {
+  return input ? isAfter(parseISO(input), parseISO(date)) : false;
+};
+
+export const isLessThanDate = (input: string, date: string) => {
+  return input ? isBefore(parseISO(input), parseISO(date)) : false;
+};
+
+export const isEqualToDate = (input: Maybe<string>, date: string) => {
+  if (!input) return false;
+  return input ? isEqual(parseISO(input), parseISO(date)) : false;
```

Comment:
Since we're already validating if input exists, this can be updated to just:
```suggestion
  return isEqual(parseISO(input), parseISO(date));
```